cdk-sfn-s3/example-pattern.json

```diff
@@ -0,0 +1,54 @@
+{
+  "title": "AWS Step Functions to Amazon S3",
+  "description": "Creates a Step Functions workflow that puts an object to an S3 bucket.",
+  "language": "TypeScript",
+  "level": "200",
+  "framework": "CDK",
+  "introBox": {
+    "headline": "How it works",
+    "text": [
+      "Step Functions receives an execution request using the `start-execution` API command and puts an object to an S3 bucket using the `PutObject` API call from the AWS SDK service integration."
+    ]
+  },
+  "gitHub": {
+    "template": {
+      "repoURL": "https://github.com/aws-samples/serverless-patterns/tree/main/cdk-sfn-s3",
+      "templateURL": "serverless-patterns/cdk-sfn-s3",
+      "projectFolder": "cdk-sfn-s3",
+      "templateFile": "src/lib/cdk-sfn-s3-stack.ts"
```

Comment:
Running into an error on validating this file. Can you please remove the projectFolder value from the templateFile?

Simple fix would be to rename the `cdk-sfn-s3-stack.ts` to something else and update the `templateFile` value